client/src/utils/BGraphUtil.ts

```diff
@@ -220,26 +207,71 @@ export const filterToBgraph = (bgraph: any, filters: Filters): any => {
       clauses.push(QUERY_FIELDS_MAP.BIO_EDGE_POST as unknown as Filter);
     }
 
+    let hasPathFilters = false;
+    for (const clause of clauses) {
+      if (filterTermToPriorityRank[clause.field] === PATH_PRIORITY_RANK) {
```

Comment:
i would probably have done

```
clause.field === QUERY_FIELDS_MAP.BIO_PATH.field
```

because the priority rank is meant to be used for priority rank only and is not meant to be used as an ID. but this works too and theres no real not to do it this way so this is up to you as well.